packages/vats/scripts/init-core.js

```diff
@@ -0,0 +1,30 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+
+export const defaultProposalBuilder = async ({ publishRef, install }) =>
```

Comment:
is "default" here meant like "default export" or are there expected to be multiple proposal builders in a module like this?